relayer/src/service.ts

```diff
@@ -105,11 +107,21 @@ export class MessageRelayerService extends BaseServiceV2<
   }
 }
 
+async function returnOnPpidChange(start_ppid: number) {
+  while (start_ppid === ppid) {
+    await sleep(500);
+  }
```

Comment:
Should we maybe log something in this case?

Reply:
Good point, will add some log.